docs/source/guide/explanation/algorithms/visual_prompting/index.rst

```diff
@@ -0,0 +1,101 @@
+Visual Prompting
+=================
+
+Visual prompting is a computer vision task that uses a combination of an image and prompts, such as texts, bounding boxes, points, and so on to troubleshoot problems.
+Using these useful prompts, the main purpose of this task is to obtain labels from unlabeled datasets, and to use generated label information on particular domains or to develop a new model with the generated information.
+
+This section examines the solutions for visual prompting offered by the OpenVINO Training Extensions library.
+`Segment Anything (SAM) <https://arxiv.org/abs/2304.02643>`_, is one of the most famous visual prompting methods and this model will be used to adapt a new dataset domain.
+Because `SAM <https://arxiv.org/abs/2304.02643>`_ was trained by using web-scale dataset and has huge backbone network, fine-tuning the whole network is difficult and lots of resources are required.
+Therefore, in this section, we try to fine-tune only mask decoder only for several epochs to increase performance on the new dataset domain.
```

Comment:
```suggestion
Therefore, in this section, training the whole network will not be handled
```

Reply:
I think your previous intention is to introduce the fine-tune the mask decoder, if so, how about introducing the CLI example?

Reply:
The default setting is fine-tuning only the mask decoder and we cannot choose which modules will be fine-tuned in CLI.
It is possible in `config.yaml`, but introducing modifying `config.yaml` to fine-tune other modules is not in this section's scope.
So, we don't need to introduce CLI example.

Reply:
Hmm, if so, I think  this "Therefore, in this section, training the whole network will not be handled" sentence also need to be deleted since we didn't guide the fine-tune in this section.

Reply:
Yes, and I described "fine-tuning the whole network is difficult and lots of resources are required" in the above line, so "fine-tune only mask decoder only" should be there.

Reply:
Then "Therefore, finetuning mask decoder is only supported." is a proper sentence that removes the "in this section", I think. I couldn't find any `try` in this section.